src/services/constants.service.ts

```diff
@@ -7,7 +7,8 @@ export class ConstantsService {
     static readonly disableFaviconKey: string = 'disableFavicon';
     static readonly disableAutoTotpCopyKey: string = 'disableAutoTotpCopy';
     static readonly enableAutoFillOnPageLoadKey: string = 'enableAutoFillOnPageLoad';
-    static readonly lockOptionKey: string = 'lockOption';
+    static readonly vaultTimeoutKey: string = 'vaultTimeout';
```

Comment:
This will break people's existing saved settings. We have to be backwards compat here. We can change the variable name, but not the string value.

Reply:
I've currently accounted for that within the sub-classed storageService class (specific to each platform).

The `init` call will look for the legacy `lockOption` key and use that value if a value for the new `vaultTimeout` key is not found (aka, they haven't saved it yet).

```
        // Handle legacy conversion to new storage key
        const legacyLockOption = await this.get<number>('lockOption');
        const vaultTimeout = await this.get<number>(ConstantsService.vaultTimeoutKey);
        if (vaultTimeout == null && !this.platformUtilsService.isDev()) {
            await this.save(ConstantsService.vaultTimeoutKey, legacyLockOption == null ? 15 : 
                  legacyLockOption);
        }
```

Can obviously do either way - just wanted to share what my thoughts were before confirming this change.

Reply:
You will have to implement that on each platform, which is going to be a little messy. Let's just leave the old `lockOption` string for now. We can always clean up key strings later if we decide to.

Reply:
Sounds good sir. Thanks!